NoteworthyUserManual.md

```diff
@@ -0,0 +1,46 @@
+# Description 
+Welcome to Noteworthy! We are the #1 Notetaking Application For Markdown files. Some of our current available functions >>> include writing, viewing, searching, listing, deleting, clearing, and converting to PDF. If you have any questions relating to Markdown the link below provides useful information
+[Markdown Guide](https://www.markdownguide.org/)
+
+# Set Up/Installation 
+You can access the required code for the Noteworthy application using the link at the bottom of this section. Furthermore, it is worth noting that this is a command line application so no external downloads are required. Finally, you need to ensure you have a "Notes" subfolder inside of your Noteworthy directory.
+[Noteworthy Code](https://github.com/augustus-thomas/noteworthy)
+
```

Comment:
Can you add this?

You can install noteworthy from the command line by running
```
git clone https://github.com/augustus-thomas/noteworth
```